dashboard/consts.py

```diff
@@ -0,0 +1,10 @@
+DASHBOARD_AGENT_PORT_PREFIX = "DASHBOARD_AGENT_PORT_PREFIX:"
+DASHBOARD_AGENT_LOG_FILENAME = "dashboard_agent"
+DASHBOARD_AGENT_CHECK_PARENT_INTERVAL_SECONDS = 2
+UPDATE_NODES_INTERVAL_SECONDS = 5
```

Comment:
Can you set it to 1 second? We'd like to make it real time. It'll be also great if this value is configurable.

Reply:
OK. I will make the values configurable in the future. Here are two ways:

- Read value from environment, just like `env_integer` in `ray_constants.py`.
- Pass value from dashboard arguments.

I prefer the first one, It's cleaner but not easy to use.

Reply:
I think the first one is fine. It'll be pretty easy to iterate or upgrade if necessary.